physionet-django/user/forms.py

```diff
@@ -585,7 +585,7 @@ def clean(self):
                        data['reference_organization'],
                        data['reference_title']]
 
-        ref_required = data['researcher_category'] in [0, 1, 6, 7]
+        ref_required = data['researcher_category'] in range(8)
```

Comment:
If everyone requires a reference, why check against range at all? Why not just set to True?

Reply:
Thanks Tom good point. I can update ref_required=True

We would need to update text for the text. It is currently:

The following steps will help to ensure a timely review of your application:
- Listing a reference is always helpful. Remember to prompt your reference to expect an email about your application.
- It is also helpful if your reference is a registered user and they have linked their ORCID iD to their profile. Please provide the email associated with their account.

The following steps will help to ensure a timely review of your application:
- All users should list a reference. Remember to prompt your reference to expect an email about your application.
- It is also helpful if your reference is a registered user and they have linked their ORCID iD to their profile. 

I don't think we need "Please provide the email associated with their account." as we required an institutional/academic in the other PR.

Let me know if that works!

Reply:
@lamawmouk 

The following change sounds good to me:

> All users should list a reference. Remember to prompt your reference to expect an email about your application.

I'm not sure I agree with the following comment. Wouldn't we still prefer to have the email address linked with the PhysioNet account? (e.g. otherwise how would we find the linked ORCID?).

> I don't think we need "Please provide the email associated with their account." as we required an institutional/academic in the other PR.

Reply:
> I'm not sure I agree with the following comment. Wouldn't we still prefer to have the email address linked with the PhysioNet account? (e.g. otherwise how would we find the linked ORCID?).

I guess they may end up in a conflicted state (use gmail connected to account or institutional address?) so I can see why you are suggesting this change!